lib/purchases_flutter.dart

```diff
@@ -628,6 +635,32 @@ class Purchases {
   /// Android only. Call close when you are done with this instance of Purchases to disconnect
   /// from the billing services and clean up resources
   static Future<void> close() => _channel.invokeMethod('close');
+
+  static Future<Map<String, dynamic>>
+      _invokeMethodReturningPurchaserInfoJsonFromMap(
```

Comment:
Better names accepted

Reply:
could just be `invokeMethodReturningPurchaserInfoJson`. but maybe we can also just call `invokeMethodReturningMap` here, and then have a method `getPurchaserInfoJsonFromMap` or something, to separate out the two pieces.

Reply:
I liked the idea of extracting `_getPurchaserInfoJsonFromMap`. Checkout the latest commit